lib/Screens/statistics.dart

```diff
@@ -31,11 +43,463 @@ class StatisticsPage extends StatelessWidget {
                       }),
                 ],
               ),
-              Text("Statistiken")
+              Text("Statistiken"),
+              //a future builder is used to deal with the asynchronos behavior of _makeLevelTile()
+              FutureBuilder(
+                  future: _makeLevelTile(0.5),
+                  builder: (context, AsyncSnapshot<Container> snapshot) {
+                    if (snapshot.hasData) {
+                      //snapshot.data is the widget of the future returned by _makeLevelTile()
+                      return snapshot.data;
+                    } else {
+                      return CircularProgressIndicator();
+                    }
+                  }),
+              Expanded(
+                child: GridView.count(
+                  padding: const EdgeInsets.all(20),
+                  crossAxisCount: 1,
+                  children: [
+                    //a future builder is used to deal with the asynchronos behavior of _makeFavBoxTile()
+                    FutureBuilder(
+                        future: _makeFavBoxTile(),
+                        builder: (context, AsyncSnapshot<Card> snapshot) {
+                          if (snapshot.hasData) {
+                            //snapshot.data is the widget of the future returned by _makeFavBoxTile()
+                            return snapshot.data;
+                          } else {
+                            return Card(
+                                child: Center(
+                                    child: Text(
+                                        "Du hast bisher noch keine Box besucht!")));
+                          }
+                        }),
+                    //_makeDataTile(45),
+                    _makeTaskTile(),
+                    _makeMemoryTile(),
+                    GestureDetector(
+                        onTap: () {
+                          Hive.openBox('storage').then((storage) {
+                            Stats.reset(storage);
+                          });
```

Comment:
normally you would always have to close boxes after opening them up.  Best would be to open it up in init-state once and closing it in dispose.